src/api/surahs.ts

```diff
@@ -0,0 +1,36 @@
+import axios, { AxiosResponse } from 'axios';
+import buildUrl from './buildUrl';
+
+export async function getSurahs(): Promise<AxiosResponse> {
+  try {
+    return await axios.get(buildUrl('/surahs'));
```

Comment:
To make this really useful, I think we want to cast the response object to the types defined in this package:
https://www.npmjs.com/package/qj_madinah